### DISCUSSION TITLE
Radar Chart responsiveness issues

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [recharts/recharts#3420] [Q&A] [answered]
Hey there,

I'm having responsiveness issues in my project because when I shrink the screen, the graphic remains the same size, basically it doesn't adapt to the screen size when the screen is reduced.
Does anyone have an idea how to resolve this?

A simple example of my problem: https://bitbucket.org/201flaviosilva-labs/react-rechart-chart-width/

Thanks for any help :)

(Solution here: https://bitbucket.org/201flaviosilva-labs/react-rechart-chart-width/commits/dcca97e4e7766bf0469f12dfb7dc69a3b39dbe23)

**Comment by ckifer** [comment, marked as answer]
Hi!

There's a few issues out there discussing this. 

Please see recent PR https://github.com/recharts/recharts/pull/3391 (not released yet, old issue #172 linked in PR). 

Essentially the issue is with how min-width: auto works. Specifying minWidth: 0 for your flex children should resolve the issue

**Reply by 201flaviosilva** [reply]
Ohhh, thanks, that solves 😁
